cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2744,6 +2730,19 @@
   </property>
 
   <property>
+    <name>security.authorization.propagate.privileges</name>
+    <value>true</value>
```

Comment:
I think it might be better to leave it as true for now. For one, it is not working right now because of https://issues.cask.co/browse/CDAP-11497. We can't grant privileges to some system datasets like app.meta. Also, this seems like a major behavior change in a minor release.